src/backtrace.cc

```diff
@@ -0,0 +1,5 @@
+#include "src/base/logging.cc"
```

Comment:
And CI is failing on smartos with `ld: fatal: symbol 'v8::base::DumpBacktrace()' is multiply-defined`: https://ci.nodejs.org/job/node-test-commit-smartos/2478/nodes=smartos14-32/console